src/services/jsDoc.ts

```diff
@@ -1,48 +1,92 @@
 /* @internal */
 namespace ts.JsDoc {
     const jsDocTagNames = [
+        "abstract",
+        "access",
+        "alias",
+        "arg",
+        "argument",
+        "async",
         "augments",
         "author",
-        "argument",
         "borrows",
         "callback",
         "class",
+        "classDesc",
+        "const",
         "constant",
         "constructor",
         "constructs",
+        "copyright",
         "default",
         "deprecated",
         "description",
+        "emits",
+        "enum",
         "event",
         "example",
+        "exports",
         "extends",
+        "external",
         "field",
+        "file",
         "fileOverview",
+        "fires",
+        "func",

```

Comment:
so what is the soruce of these tags? it is kinda weird to have `func` and `function`.. do we need both? are both commonly used?

Reply:
The source of the tags is of course the http://usejsdoc.org/ page, `func` as well as `method` are synonyms to `function`, they are displayed between "(" and ")". I am testing the behavior in VSCode, from what I see they are parsed correctly, but there is no autocomplete for them.